test/k8sT/Policies.go

```diff
@@ -242,6 +243,16 @@ var _ = Describe("K8sPolicyTest", func() {
 			res.ExpectFail("Unexpected connection from %q to 'http://%s/private'",
 				appPods[helpers.App3], clusterIP)
 
+			// Update existing policy on port 80 from http to kafka
```

Comment:
TBH, I would add a different test here. Test is important enough to be in another test, so it's easy to run, and clean when the test fails. 

Something like: `It("Change L7 parser type")`

Reply:
That will take a lot longer to run. The test covers a special case that typically only happens on user error. I've updated the test failure reason to be more explicit in case it fails.

Reply:
No, the deploy of pods are in the BeforeAll so longer will be less than 5 seconds. If the test are loger, involve harder debug and it is difficult to understand what is tested. IMHO.

Reply:
I doubt that it only adds 5 seconds. We need to load a policy which will regenerate all endpoints as it jumps to L7 enforcement. We then delete the CNP which will cause all endpoints to be regenerated again to remove the L7 redirects.

What makes this hard to debug? It actually has a very clear error message that immediately tells what *exactly* is not working.

Reply:
For me longest test are more harder to debug, happy to keep as it is.